client/src/Components/Layouts/Customers/index.jsx

```diff
@@ -1,20 +1,134 @@
 import React, { Component } from 'react';
-import Sidebar from '../../CommonComponent/Sidebar';
-export default class Customers extends Component{
-    state={
-        customers:[],
-        name:''
+import './style.css';
+import { Input } from 'antd';
+import Table from '../../CommonComponent/Table/Table';
+import Button from '../../CommonComponent/Button';
+import Sidebar from "../../CommonComponent/Sidebar/index";
+import Header from "../../CommonComponent/Header/index";
+import Navbar from "../../CommonComponent/Navbar/index"
+import { viewPopup, editPopup, deletePopup } from "../../CommonComponent/Table/Popups";
+import { Icon } from 'antd';
+import { DatePicker } from 'antd';
+const { RangePicker } = DatePicker;
+
+const Search = Input.Search;
+
+
+
+export default class Customers extends Component {
+    state = {
+        customers: [{
```

Comment:
delete costumer objects, so after adding the back end side and fetching the data you will not need it.